fan-xiaomi.js

```diff
@@ -364,7 +437,11 @@ Natural
         // Timer
         let timer_display = '0m'
         if(delay_off_countdown) {
-            let total_mins = delay_off_countdown / 60
+            let total_mins = delay_off_countdown
+            if (model !== 'dmaker.fan.p5') {
```

Comment:
I believe this block is now unnecessary with your changes at https://github.com/syssi/xiaomi_fan/pull/57 where the service call now accepts minutes.

Reply:
This is needed as it is. p5 entity attribute attrs['delay_off_countdown'], contains data in minutes, all others contain data in seconds. This block part is needed for UI representation of time, that is left on countdown timer, and not for service call to xiaomi_fan.

Reply:
My bad I mixed this up with the service call, thanks for pointing it out!